src/tests/dbus-tests/test_60_partitioning.py

```diff
@@ -95,7 +95,7 @@ def test_create_mbr_partition(self):
         sys_num = int(self.read_file('%s/partition' % part_syspath))
         dbus_num.assertEqual(sys_num)
 
-    def test_create_extended_partition(self):
+    def create_extended_partition(self, ext_options, log_options, fstype=''):
```

Comment:
It's not ``fstype``, it is the partition type (see the [docs](http://storaged.org/doc/udisks2-api/latest/gdbus-org.freedesktop.UDisks2.Partition.html#gdbus-property-org-freedesktop-UDisks2-Partition.Type) for details)